frontend/src/lib/lemon-ui/LemonTree/LemonTree.tsx

```diff
@@ -290,15 +287,112 @@ const LemonTreeNode = forwardRef<HTMLDivElement, LemonTreeNodeProps>(
                                                         enableDragAndDrop &&
                                                         isItemDraggable?.(item) &&
                                                         item.record?.path
-                                                            ? (button) => (
-                                                                  <TreeNodeDraggable
-                                                                      id={item.record?.path}
-                                                                      enableDragging
-                                                                      className="h-[var(--button-height-base)]"
-                                                                  >
-                                                                      {button}
-                                                                  </TreeNodeDraggable>
-                                                              )
+                                                            ? (button) => {
+                                                                  // Check if multi-selection is enabled and this item is checked
+                                                                  const hasCheckedItems =
+                                                                      enableMultiSelection &&
+                                                                      checkedItemIds?.includes(item.id)
```

Comment:
This might be easier to get from `item.checked` thought not a big difference. If checkedItemIds gets huge this `includes` check might get slow if ran many times (e.g. for all files), but that's not really a worry now